### PR TITLE
Simulcast support

### PR DESCRIPTION
**Pull request by jianjunz**
- Publish method accepts `RTCRtpEncodingParameters` as options.
- Upgrade conference protocol version to 1.1.

### CODE REVIEW

File: src/sdk/conference/subscription.js [hunk 1]
```diff
@@ -157,6 +157,13 @@ export class VideoSubscriptionConstraints {
      * @desc Only keyFrameIntervals listed in Owt.Conference.VideoSubscriptionCapabilities are allowed.
      */
     this.keyFrameInterval = keyFrameInterval;
+    /**
+     * @member {?number} rid
+     * @instance
+     * @memberof Owt.Conference.VideoSubscriptionConstraints
+     * @desc Restriction identifier to identify the RTP Streams within an RTP session. When rid is specified, other constraints will be ignored.
+     */
+    this.rid = rid;
```
Comment (starwarfan) [inline src/sdk/conference/subscription.js]: rid is undefined here

Comment (jianjunz) [reply src/sdk/conference/subscription.js]: Thanks. Fixed.

File: src/sdk/conference/client.js [hunk 2]
```diff
@@ -394,9 +394,10 @@ export const ConferenceClient = function(config, signalingImpl) {
    * @desc Publish a LocalStream to conference server. Other participants will be able to subscribe this stream when it is successfully published.
    * @param {Owt.Base.LocalStream} stream The stream to be published.
    * @param {Owt.Base.PublishOptions} options Options for publication.
+   * @param {string} videoCodecs Video codec names for publishing. Valid values are 'VP8', 'VP9' and 'H264'. This parameter only valid when options.video is RTCRtpEncodingParameters. Publishing with RTCRtpEncodingParameters is an experienmental feature. This paramter is subject to change.
```
[the remark this answers is not in the document]
Comment (jianjunz) [reply src/sdk/conference/client.js]: Fixed.

File: src/sdk/base/publication.js [hunk 3]
```diff
@@ -181,14 +181,14 @@ export class PublishOptions {
...
-     * @desc Parameters for audio RtpSender.
+     * @desc Parameters for audio RtpSender. Publishing with RTCRtpEncodingParameters is an experienmental feature. It is subject to change.
      */
     this.audio = audio;
     /**
      * @member {?Array<Owt.Base.VideoEncodingParameters> | ?Array<RTCRtpEncodingParameters>} video
      * @instance
      * @memberof Owt.Base.PublishOptions
-     * @desc Parameters for video RtpSender.
+     * @desc Parameters for video RtpSender. Publishing with RTCRtpEncodingParameters is an experienmental feature. It is subject to change.
```
[the remark this answers is not in the document]
Comment (jianjunz) [reply src/sdk/base/publication.js]: Fixed.

File: src/sdk/base/publication.js [hunk 4]
```diff
@@ -181,14 +181,14 @@ export class PublishOptions {
      * @member {?Array<Owt.Base.AudioEncodingParameters> | ?Array<RTCRtpEncodingParameters>} audio
      * @instance
      * @memberof Owt.Base.PublishOptions
-     * @desc Parameters for audio RtpSender.
+     * @desc Parameters for audio RtpSender. Publishing with RTCRtpEncodingParameters is an experienmental feature. It is subject to change.
```
[the remark this answers is not in the document]
Comment (jianjunz) [reply src/sdk/base/publication.js]: Fixed.

File: src/sdk/conference/client.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jianjunz) [reply src/sdk/conference/client.js]: Yes, it's an array of string.

[the remark this answers is not in the document]
Comment (jianjunz) [reply src/sdk/conference/client.js]: It's actually an array of `Owt.Base.VideoCodec`. But I found `Owt.Base.VideoCodec` is not accessible outside SDK.

**Pull request merged**